npm-shrinkwrap.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "hipache",
+  "version": "0.4.0",
```

Comment:
this PR is against master, it just uses the version from package.json. i've got another branch for the 0.3.x release, but there's no branch to submit a PR against, so it's idling over on my fork for now.